ocrd_models/ocrd_page_user_methods/id.py

```diff
@@ -0,0 +1,5 @@
+@property
+def id(self):
+    if hasattr(self, 'pcGtsId'):
+        return self.pcGtsId
```

Comment:
That can still fail, though: If no `@pcGtsId` is annotated, then this would return `None`, which is not hashable.

```suggestion
        return self.pcGtsId or ''
```